deb-x64/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM ubuntu as CURL_GETTER
 RUN apt-get update && apt-get install -y wget
 RUN wget https://github.com/moparisthebest/static-curl/releases/download/v7.79.1/curl-amd64
 
-FROM debian:wheezy-backports
+FROM debian:jessie-backports
```

Comment:
Should we keep building both the Wheezy and Jessie versions until we release 7.36? We might potentially have to rebuild the Debian Wheezy image if we have to do a 7.34.x release, right?